### PR TITLE
LUN-152: Fix audio files

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/components/AudioPlayer.tsx [hunk 1]
```diff
@@ -15,14 +16,19 @@ const StyledView = styled.View`
...
   width: 40px;
   height: 40px;
   border-radius: 50px;
-  background-color: ${props => props.theme.colors.lunesFunctionalIncorrectDark};
+  background-color: ${props =>
+    props.disabled
+      ? props.theme.colors.lunesBlackUltralight
+      : props.isActive
+      ? props.theme.colors.lunesRed
+      : props.theme.colors.lunesRedDark};
```
Comment (klinzo) [inline src/components/AudioPlayer.tsx:31]: You are doing it as well :astonished: Nested ternary expressions, tzzz :smile:

**Review by klinzo — COMMENTED** [review]
Playing audio files does not work at all if there is no tts engine available (which is the case for me). I think it also makes no sense to initialize both all the time. I created a PR hopefully fixing the issue with no tts engine which also tidies stuff up and only initializes the necessary audio: #149

File: src/components/AudioPlayer.tsx [hunk 2]
```diff
@@ -83,7 +89,10 @@ const AudioPlayer = (props: AudioPlayerProps): ReactElement => {
     if (isInitialized) {
       setIsActive(true)
       if (audio) {
-        SoundPlayer.playUrl(document?.audio)
+        SoundPlayer.loadUrl(audio)
+        _onFinishedLoadingSubscription = SoundPlayer.addEventListener('FinishedLoadingURL', ({ success, url }) => {
+          SoundPlayer.play()
+        })
```
Comment (klinzo) [inline src/components/AudioPlayer.tsx:95]: Not necessary to do this each time you play an audio.

File: src/components/AudioPlayer.tsx (hunk 1 above)
Comment (SteffiStuffel) [reply src/components/AudioPlayer.tsx:30]: Its not mine, i just copied :smile:

**Pull request merged**